tests/unit/modules/test_cluster_role_info.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import pytest
+
+from ansible_collections.sensu.sensu_go.plugins.module_utils import (
+    errors, utils,
+)
+from ansible_collections.sensu.sensu_go.plugins.modules import cluster_role_info
+
+from .common.utils import (
+    AnsibleExitJson, AnsibleFailJson, ModuleTestCase, set_module_args,
+)
+
+
+class TestClusterRoleInfo(ModuleTestCase):
+    def test_get_all_roles(self, mocker):
```

Comment:
I would rename the method to `test_get_all_cluster_roles`, just to be consistent with naming in `test_cluster_role.py`.